public/index.html

```diff
@@ -77,7 +77,6 @@
       }
 
       #background-radial-gradient {
```

Comment:
can you add a comment explaining usage? the id name reads like a misnomer now that there's no background info at all in the css and it's all applied once js runs